gateway/api/schedule.py

```diff
@@ -23,6 +23,43 @@
 logger = logging.getLogger("commands")
 
 
+def _configure_job_to_use_gpu(job: Job):
+    gpujobs = create_gpujob_allowlist()
+    if (
+        job.program.provider
+        and job.program.provider.name in gpujobs["gpu-functions"].keys()
+    ):
+        logger.debug("Job [%s] will be run on GPU nodes", job.id)
+        job.gpu = True
+        job.save()
```

Comment:
After reviewing the PR with @korgan00 we are pretty sure that we can remove this `save` as it's being saved in the `schedule_queued_jobs` command.

Reply:
Solved in: [6cbffa8](https://github.com/Qiskit/qiskit-serverless/pull/1664/commits/6cbffa8ac8953e47452ed116aadc137aff12c443)